shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -202,7 +204,7 @@ mkNonceFromNumber :: Word64 -> Nonce
 mkNonceFromNumber =
   Nonce
     . (castHash :: Hash SHA256 Word64 -> Hash SHA256 Nonce)
-    . hash --TODO: use simpler serialisation
+    . hashRaw (BSL.toStrict . B.runPut . B.putWord64be)
```

Comment:
This hashes the word64 in network byte order instead of using a cbor encoding.

Reply:
:clap: